pkgs/development/python-modules/anywidget/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pythonOlder
+, hatch-jupyter-builder
+, hatchling
+, importlib-metadata
+, ipywidgets
+, jupyterlab
+, psygnal
+, typing-extensions
+, watchfiles
+}:
+
+buildPythonPackage rec {
+  pname = "anywidget";
+  version = "0.6.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
I would have written the properties instead of inherit them. It has been one of my review last time. (It's fine in my opinion)

Reply:
Good point.
We prefer to avoid `repo = pname` in `fetchFromGitHub` because they are not necessarily the same.
I mean, `repo` and `pname` are semantically different.
However, pname [should be the same](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-contributing) for python packages as PyPI's pname.
The pname in buildPythonPackage and the pname in fetchPypi are essentially the same things.